mk/test.mk

```diff
@@ -7,7 +7,7 @@ GINKGO_UNIT_TEST_FLAGS ?= \
 
 # -race requires CGO_ENABLED=1 https://go.dev/doc/articles/race_detector and https://github.com/golang/go/issues/27089
 UNIT_TEST_ENV=$(GOENV) CGO_ENABLED=1 KUBEBUILDER_ASSETS=$(KUBEBUILDER_ASSETS) TMPDIR=/tmp UPDATE_GOLDEN_FILES=$(UPDATE_GOLDEN_FILES) $(if $(CI),TESTCONTAINERS_RYUK_DISABLED=true,GINKGO_EDITOR_INTEGRATION=true)
-GINKGO_TEST:=$(GINKGO) $(GOFLAGS) $(LD_FLAGS) -p --keep-going --keep-separate-reports --junit-report results.xml --output-dir $(REPORTS_DIR)
+GINKGO_TEST:=$(GINKGO) $(GOFLAGS) $(LD_FLAGS) -p --keep-going --junit-report results.xml --output-dir $(REPORTS_DIR)
```

Comment:
keep-separate-reports causes files to be named with test prefix like `build/reports/test_e2e_env_kubernetes_results.xml`. Traced this parameter back to https://github.com/kumahq/kuma/commit/5ee8cb75a88cf33c022e30d2231f98363f8af924#diff-f5975eb03c05409935280a8b9d6538a1d61b21310990e82b24c5121279e23cefR34 - @bartsmykla do you remember why you added it?

Reply:
I unfortunately don't remember